src/main/java/com/google/ehub/data/EntertainmentItem.java

```diff
@@ -39,4 +52,96 @@ public String getDescription() {
   public String getImageUrl() {
     return imageUrl;
   }
+
+  public String getReleaseDate() {
+    return releaseDate;
+  }
+
+  public String getRuntime() {
+    return runtime;
+  }
+
+  public String getGenre() {
+    return genre;
+  }
+
+  public String getDirectors() {
+    return directors;
+  }
+
+  public String getWriters() {
+    return writers;
+  }
+
+  public String getActors() {
+    return actors;
+  }
+
+  public static final class Builder {
+    private Optional<Long> uniqueId = Optional.empty();
+    private String title = "";
+    private String description = "";
+    private String imageUrl = "";
+    private String releaseDate = "";
+    private String runtime = "";
+    private String genre = "";
+    private String directors = "";
+    private String writers = "";
+    private String actors = "";
+
+    public EntertainmentItem build() {
+      return new EntertainmentItem(uniqueId, title, description, imageUrl, releaseDate, runtime,
+          genre, directors, writers, actors);
+    }
+
+    public Builder setUniqueId(Optional<Long> uniqueId) {
```

Comment:
this method can just be setUniqueId(long uniqueId) then build() handles optional.of()

Reply:
OK I replaced Optional parameter in the method used by the builder class and updated all function calls that used this method in last commit.